data/scenarios/blank.yaml

```diff
@@ -0,0 +1,15 @@
+version: 1
+name: Blank world
+description: No constraints and no obstacles!  Do whatever you want, in this empty void!
+creative: true
+robots:
+  - name: base
+    loc: [0,0]
+    dir: [0,1]
+    heavy: true
+    display:
+      char: Ω
+      attr: robot
+world:
+  default: [grass]
+  offset: true
```

Comment:
Should be able to remove `offset: true`, it does nothing if the world is blank (`offset: true` means we try to find a starting location for the base with various starting resources nearby).

Reply:
Also, maybe the default should be `blank` instead of `grass`?  After all, the world is supposed to be... blank. :smile:

Reply:
Of course, we need to add a way to change the terrain... #742 .

Reply:
Sure, done! I purposely did not make it entirely blank... at least for now. There are a couple of reasons:
1. There are more games with default flat green world, like [Superflat in Minecraft](https://minecraft.fandom.com/wiki/Superflat). The first programming game I played was a gardener robot puzzle, so that might have influenced me.
2. It's harder to tell apart from the side panels.
3. A truly blank void is depressing.